### PR TITLE
Create Converter pallet on Parachain#111

### PR DESCRIPTION
**Pull request by N1ghtStorm**
Converter pallet adds direct conversion between AssetId(basically it is [u8; 32] array as it done on Sora Mainnet) and xcm Multilocation.

### CODE REVIEW

File: common/src/primitives.rs [hunk 1]
```diff
@@ -28,98 +28,4 @@
...
-	// SORA
-	XOR,
-	// Polkaswap
-	PSWAP,
-	//  Validators
-	VAL,
-	// XOR Synthetic Dolla
-	XSTUSD,
-}
+pub type AssetId = [u8; 32];
```
Comment (wer1st) [inline common/src/primitives.rs]: Add empty line

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -109,7 +110,7 @@ std = [
 	"pallet-collator-selection/std",
 	"pallet-session/std",
 	"pallet-sudo/std",
-	"pallet-template/std",
+	"pallet-converter/std",
```
Comment (wer1st) [inline runtime/Cargo.toml]: Move it above, after line 110. It looks like to follow alphabetical order

File: pallets/converter/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", features = ["derive"], default-features = false }
+scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
+
+# Substrate
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", default-features = false, optional = true, branch = "polkadot-v0.9.24" }
+frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.24" }
+frame-system = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.24" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.24" }
```
Comment (wer1st) [inline pallets/converter/Cargo.toml:22]: We use branch = "polkadot-v0.9.25" in mainnet

**Review by wer1st — CHANGES_REQUESTED** [review]

**Comment by wer1st** [comment]
Where is weights.rs ?

**Comment by N1ghtStorm** [comment]
> Where is weights.rs ?

the extrinsics are not benchmarked yet

**Comment by wer1st** [comment]
> > Where is weights.rs ?
> 
> the extrinsics are not benchmarked yet

Then just run local benchmark to fill it. Values will be replaced later.

File: pallets/converter/Cargo.toml (hunk 3 above)
Comment (N1ghtStorm) [reply pallets/converter/Cargo.toml:22]: This pallet made for parachain, that is 24 version

**Pull request merged**